src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/config/HmppsIncentivesApiExceptionHandler.kt

```diff
@@ -69,7 +71,7 @@ class HmppsIncentivesApiExceptionHandler {
   }
 
   @ExceptionHandler(ParameterValidationException::class)
-  fun handleValidationException(e: ParameterValidationException): ResponseEntity<ErrorResponse> {
+  fun handleParameterValidationException(e: ParameterValidationException): ResponseEntity<ErrorResponse> {
```

Comment:
It might not have been a problem due to annotation, but this method was not uniquely names